.github/workflows/scripts/check-if-modified.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+#https://git-scm.com/docs/git-diff#Documentation/git-diff.txt-emgitdiffemltoptionsgt--ltpathgt82308203
+if [[ $(git diff HEAD^ HEAD -- $1) != '' ]]; then
+    # modified
+    echo 'true'
+else
+    # unmodified
+    echo 'false'
+fi
```

Comment:
Nit: this can probably be inlined if it's relatively straightforward and only used in one location?